src/compiler/scala/tools/nsc/backend/jvm/BCodeBodyBuilder.scala

```diff
@@ -632,6 +633,10 @@ abstract class BCodeBodyBuilder extends BCodeSkelBuilder {
             case _ =>
               abort(s"Cannot instantiate $tpt of kind: $generatedType")
           }
+        case Apply(_, args) if app.hasAttachment[delambdafy.LambdaMetaFactoryCapable] =>
+          val attachment = app.attachments.get[delambdafy.LambdaMetaFactoryCapable].get
+          genLoadArguments(args, paramTKs(app))
```

Comment:
In that case, `Delambdafy` rewrites to `$anonFunFactory(receiver, arg0, ... argN)`, where `$anonFunFactory` is a fictional symbol that with a result type `(LambdaParam0, .. LambdaParam0) => LambdaResult`